Build.proj

```diff
@@ -39,7 +39,13 @@
           Inputs="@(RepoTasksSrc)"
           Outputs="$(RepoTasksOutputFile)">
     <ItemGroup>
-      <RepoTaskProjects Include="$(RepoTasksDir)**\*.csproj" />
+      <RepoTaskProjects Include="$(RepoTasksDir)\installer.tasks\**\*.csproj" />
+      <RepoTaskProjects Condition="'$(TargetOS)' == 'Android'"
+                        Include="$(RepoTasksDir)\mobile.tasks\AndroidAppBuilder\AndroidAppBuilder.csproj" />
+      <RepoTaskProjects Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS'"
+                        Include="$(RepoTasksDir)\mobile.tasks\AppleAppBuilder\AppleAppBuilder.csproj" />
+      <RepoTaskProjects Condition="'$(TargetsMobile)' == 'true'"
+                        Include="$(RepoTasksDir)\mobile.tasks\AotCompilerTask\MonoAOTCompiler.csproj" />
```

Comment:
I think there's a better way to do this without loosing the glob over the whole repotasksdir. Let me quickly try something out.